packages/core/plugins/core-plugins-server-internal/src/discovery/plugin_manifest_from_plugin_package.ts

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { snakeCase } from 'lodash';
+import { PluginPackageManifest } from '@kbn/repo-packages';
+import { PluginManifest } from '@kbn/core-plugins-server';
+import { PluginType } from '@kbn/core-base-common';
+
+export function pluginManifestFromPluginPackage(
+  kibanaVersion: string,
+  manifest: PluginPackageManifest
+): PluginManifest {
+  return {
+    type: manifest.plugin.type === 'preboot' ? PluginType.preboot : PluginType.standard,
+    id: manifest.plugin.id,
+    version: '1.0.0',
```

Comment:
When we proposed the new package manifest schema we proposed dropping the version, and have done so. This is here as a placeholder because this is a required string on the existing manifest type. The version of all packages in the Kibana repository are equal to Kibana's version, there's no reason for them all to have individual versions. Third party plugins are currently limited to the existing/legacy plugin interface so they still have real versions for now and we may maintain version support for third party plugins to some extent.

As for `devOnly` packages, they will be excluded by the build process so they won't be of concern in this context. If we have a `devOnly` plugin then it would only be something that could be enabled locally when running from source. It wouldn't be available in CI or any built version of Kibana. As such I don't expect any to ever exist.

Reply:
Ah, that makes sense. Thanks!